src/main/msp/msp.c

```diff
@@ -2130,7 +2130,9 @@ static mspResult_e mspFcProcessOutCommandWithArg(mspDescriptor_t srcDesc, int16_
             sbufWriteU8(dst, currentPidProfile->simplified_i_gain);
             sbufWriteU8(dst, currentPidProfile->simplified_pd_ratio);
             sbufWriteU8(dst, currentPidProfile->simplified_pd_gain);
+#ifdef USE_D_MIN
             sbufWriteU8(dst, currentPidProfile->simplified_dmin_ratio);
+#endif
```

Comment:
These will have to include sending / receiving an equal number of buffer bytes to make sure that the format of the MSP message stays the same independent of the features built into the firmware.